src/helper/storage/handler.js

```diff
@@ -3,9 +3,12 @@ import DummyStorage from './dummy';
 import CookieStorage from './cookie';
 import Warn from '../warn';
 
-function StorageHandler() {
+function StorageHandler(options) {
   this.warn = new Warn({});
   this.storage = new CookieStorage();
+  if (options.__tryLocalStorageFirst !== true) {
```

Comment:
Hello @luisrudge. How can I use LocalStorage instead of CookieStorage in a TypeScript application? It seems impossible to set __tryLocalStorageFirst to `true` because of `AuthorizeOptions` or `AuthOptions` type definition: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/auth0-js/index.d.ts#L166

Reply:
send a PR to the typing repo adding this flag?

Reply:
Also, why you want to use localStorage instead of cookies? that would be pretty helpful to understand, since we're considering removing localStorage in the future.

Reply:
Hello @luisrudge

"send a PR to the typing repo adding this flag?", Yes. PR done and merged by DefinitelyTyped team 👍

"Also, why you want to use localStorage instead of cookies?" Because I did not find how to make auth0 cookie work with Electron.